python/tvm/relay/build_module.py

```diff
@@ -151,7 +151,8 @@ def optimize(func, params=None):
         func = ir_pass.combine_parallel_conv2d(func)
 
     if cfg.pass_enabled("FoldConstant"):
-        func = ir_pass.fold_constant(func)
+        with _target.create("llvm"):
```

Comment:
This new target is unnecessary because `FoldConstant` always creates a new llvm target

Reply:
BTW, can you compiler inception_v3 with relay?

Reply:
I can compile inception v3 without AlterOpLayout, otherwise I got `type_infer.cc:314: the function is provided too many arguments (nullptr)`

Reply:
Fixed